packages/dashboard/src/customization/propertiesSections/aggregationSettings/index.tsx

```diff
@@ -35,11 +37,22 @@ export const isOnlyAggregatedDataWidget = (
 ): widget is LineScatterChartWidget =>
   isOnlyAggregated.some((t) => t === widget.type);
 
+export const isAggregateAndRawDataWidget = (
+  widget: DashboardWidget
+): widget is LineScatterChartWidget =>
+  (!!localStorage?.getItem('USE_UPDATED_KPI') &&
```

Comment:
Just FYI, use of local storage requires AppSec approval, and has a pretty high bar. It may not be explicitly required to publish this library, but if we use this in an App like centurion/console, this will be an issue.